reporter/src/funTest/assets/static-html-reporter-test-input.yml

```diff
@@ -67,7 +67,7 @@ analyzer:
           - id: "Maven:org.apache.commons:commons-lang3:3.5"
       - name: "testCompile"
         dependencies:
-        - id: "Maven:junit:junit:4.12"
+        - id: "Ant:junit:junit:4.12"
```

Comment:
Why was this changed to `Ant`?

Reply:
To have a dependency that would appear before the project without this change.